cnv/cnv_users_guide/cnv-dashboard.adoc

```diff
@@ -0,0 +1,13 @@
+[id="cnv-dashboard"]
+= Using the CNV Dashboard to get cluster information
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-dashboard
+toc::[]
+
+The Web Console is part of the {CNVProductName} interface.
+
+Access the {CNVProductName} Overview Dashboard, which contains high-level information about the cluster, by clicking *Home > Dashboards > Overview* from the {product-name} Web Console.
```

Comment:
1. The variable for OCP is `{product-title}`, not `{product-name}`
2. Again, I don't think that we capitalize "web console"

Reply:
correct; web console is in lowercase

Reply:
Noted, thanks!